AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -7,6 +7,8 @@
 !
 ! SECTION: Popups - Regular rules
 !
+imdb.com#$?#div[data-testid="promptable"]:has(.ipc-rating-display-with-poster__image) { display: none !important; }
```

Comment:
I think coreLibs should correctly exclude this from `content_blocker` Filter version.

```suggestion
imdb.com#?#div[data-testid="promptable"]:has(.ipc-rating-display-with-poster__image)
```

Reply:
` { display: none !important; }` is not necessary for `#?#` rules - they won't apply on content blockers. So, this part of the rule should be removed.
@AdamWr @BlazDT could you check it on your side? I don't have the popup.

Reply:
I cannot reproduce it.

Regarding second rule:
```adblock
imdb.com#$?#body:has(div[data-testid="promptable"]:has(.ipc-rating-display-with-poster__image)) { overflow: auto !important; }
```

If I'm not wrong, it's not necessary to use `:has` two times, something like this:
```adblock
imdb.com#$?#body:has(div[data-testid="promptable"] .ipc-rating-display-with-poster__image) { overflow: auto !important; }
```
should match the same element.

Reply:
Fixed.